ci/core-dump.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -veo pipefail
+
+for file in /cores/*; do
+    echo "Core file: $file"
+    DUB_EXEC=$(echo "$file" | sed 's/\/cores\///' | sed -E 's/\.[0-9]+$//' | tr '!' '/')
+    echo "Executable: $DUB_EXEC"
+    sudo lldb -c $file -o "bt all" -o "script import os; os._exit(1)"
```

Comment:
Do we get a recent `lldb` ? They added `--batch` support too.

https://stackoverflow.com/a/36580453/1021344